qt/python/mantidqt/widgets/superplot/presenter.py

```diff
@@ -328,7 +339,10 @@ def _update_plot(self):
 
         if selection or plotted_data:
             axes.set_axis_on()
-            figure.tight_layout()
+            try:
+                figure.tight_layout()
+            except:
```

Comment:
Same comment here about catching the `ValueError`.

Reply:
done